std/math.d

```diff
@@ -4103,9 +4103,55 @@ long lrint(real x) @trusted pure nothrow @nogc
 
             return sign ? -result : result;
         }
+        else static if (F.realFormat == RealFormat.ieeeQuadruple)
+        {
+            long result;
+
+            // Rounding limit when casting from real(128-bit) to ulong.
+            enum real OF = 0x1P+112;
+
+            ulong* vl = cast(ulong*)(&x);
+
+            // Find the exponent and sign
+            ulong msb = vl[MANTISSA_MSB];
+            ulong lsb = vl[MANTISSA_LSB];
```

Comment:
I realize you're just copying the names used above in the `ieeeDouble` section, but these really should be named something else like `hi` and `lo`.

`msb` generally stands for "Most Significant Byte" (or Bit), and `lsb` for "Least Significant Byte" (or Bit). Using that acronym for 64-bit values (quad words, I think, in Intel terminology?) is confusing.